doc/changelog.rst

```diff
@@ -7,90 +7,138 @@ Motor 3.0
+- :class:`~motor.motor_tornado.MotorCollection` and :class:`motor.motor_tornado.MotorDatabase`
   now raises an error upon evaluating as a Boolean, please use the
   syntax ``if collection is not None:`` or ``if database is not None:`` as
   opposed to
   the previous syntax which was simply ``if collection:`` or ``if database:``.
   You must now explicitly compare with None.
-- :class:`~motor.MotorClient` cannot execute any operations
+- :class:`~motor.motor_tornado.MotorClient` cannot execute any operations
   after being closed. The previous behavior would simply reconnect. However,
   now you must create a new instance.
 - Empty projections (eg {} or []) for
-  :meth:`~motor.MotorCollection.find`, and
-  :meth:`~motor.MotorCollection.find_one`
+  :meth:`~motor.motor_tornado.MotorCollection.find`, and
+  :meth:`~motor.motor_tornado.MotorCollection.find_one`
   are passed to the server as-is rather than the previous behavior which
   substituted in a projection of ``{"_id": 1}``. This means that an empty
   projection will now return the entire document, not just the ``"_id"`` field.
-- :class:`~motor.MotorClient` now raises a
+- :class:`~motor.motor_tornado.MotorClient` now raises a
   :exc:`~pymongo.errors.ConfigurationError` when more than one URI is passed
   into the ``hosts`` argument.
-- :class:`~motor.MotorClient`` now raises an
+- :class:`~motor.motor_tornado.MotorClient`` now raises an
   :exc:`~pymongo.errors.InvalidURI` exception
   when it encounters unescaped percent signs in username and password when
   parsing MongoDB URIs.
-- Comparing two :class:`~motor.MotorClient` instances now
+- Comparing two :class:`~motor.motor_tornado.MotorClient` instances now
   uses a set of immutable properties rather than
-  :attr:`~motor.MotorClient.address` which can change.
+  :attr:`~motor.motor_tornado.MotorClient.address` which can change.
 - Removed the `disable_md5` parameter for :class:`~gridfs.GridFSBucket` and
   :class:`~gridfs.GridFS`. See :ref:`removed-gridfs-checksum` for details.
 - PyMongoCrypt 1.2.0 or later is now required for client side field level
   encryption support.
 
+Notable improvements
+....................
+
+- Enhanced connection pooling to create connections more efficiently and
+  avoid connection storms.
+- Added the ``maxConnecting`` URI and
+  :class:`~motor.motor_tornado.MotorClient` keyword argument.
+- :class:`~motor.motor_tornado.MotorClient` now accepts a URI and keyword
+  argument `srvMaxHosts` that limits the number of mongos-like hosts a client
+  will connect to. More specifically, when a mongodb+srv:// connection string
+  resolves to more than `srvMaxHosts` number of hosts, the client will randomly
+  choose a `srvMaxHosts` sized subset of hosts.
+- Added :attr:`motor.motor_tornado.MotorClient.options` for read-only access
+  to a client's configuration options.
+- Added support for the ``comment`` parameter to all helpers. For example see
```

Comment:
commit and let support require PyMongo 4.1 so should we bump our min version to 4.1?

Reply:
Sure, as a separate ticket?

Reply:
Yeah separately makes sense.